varya/assets/css/variables.css

```diff
@@ -153,12 +153,15 @@
 	--branding--color-link-hover: var(--global--color-primary-hover);
 	--branding--title--font-family: var(--global--font-primary);
 	--branding--title--font-size: var(--heading--font-size-h1);
+	--branding--title--font-size-mobile: calc( 0.75 * var(--heading--font-size-h1) );
```

Comment:
This is a one-off font size, which I'm not really proud of. But `--heading--font-size-h1` is too big and `--heading--font-size-h2` is too small... my comps used something right in the middle, so I stuck with that here.